pkg/schemas/stream.go

```diff
@@ -102,11 +103,9 @@ func StreamDescriptionToSchema(stream sdk.Stream) map[string]any {
 		streamSchema["source_type"] = stream.SourceType
 	}
 	if stream.BaseTables != nil {
-		idsFQN := make([]string, len(stream.BaseTables))
-		for i := range idsFQN {
-			idsFQN[i] = stream.BaseTables[i].FullyQualifiedName()
-		}
-		streamSchema["base_tables"] = idsFQN
+		streamSchema["base_tables"] = collections.Map(stream.BaseTables, func(id sdk.SchemaObjectIdentifier) string {
```

Comment:
func could be changed to `sdk.SchemaObjectIdentifeir.FullyQualifiedName` 😎

Reply:
Done.